hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -84,31 +87,32 @@ public void testSchemaConvertArray() throws IOException {
     MessageType schema = Types.buildMessage().optionalGroup().as(OriginalType.LIST).repeatedGroup()
         .optional(PrimitiveType.PrimitiveTypeName.INT32).named("element").named("list").named("int_list")
         .named("ArrayOfInts");
-
-    String schemaString = HiveSchemaUtil.generateSchemaString(schema);
+    AvroSchemaConverter avroSchemaConverter = new AvroSchemaConverter(new Configuration());
```

Comment:
guess we don't have good coverage of tests for convertFieldFromAvro. All I see are tests for testSchemaConvertArray and testSchemaConvertTimestampMicros. Would you mind adding tests for convertFieldFromAvro to cover all field types (primitives, list, enum, map, nested records)